### ISSUE TITLE
BadHeaderError at /account/invite_user/

### ISSUE THREAD

**Issue by wen2solutions** [GeoNode/geonode#3648]
I just got this error, could someone help this?

Environment:

Request Method: POST
Request URL: http://x.x.x.x/account/invite_user/

Django Version: 1.6.11
Python Version: 2.7.6
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.sites',
 'django.contrib.admin',
 'django.contrib.sitemaps',
 'django.contrib.staticfiles',
 'django.contrib.messages',
 'django.contrib.humanize',
 'django.contrib.gis',
 'pagination',
 'taggit',
 'friendlytagloader',
 'geoexplorer',
 'leaflet',
 'django_extensions',
 'autocomplete_light',
 'mptt',
 'modeltranslation',
 'djcelery',
 'pinax_theme_bootstrap_account',
 'pinax_theme_bootstrap',
 'django_forms_bootstrap',
 'account',
 'avatar',
 'dialogos',
 'agon_ratings',
 'notification',
 'announcements',
 'actstream',
 'user_messages',
 'tastypie',
 'polymorphic',
 'guardian',
 'geonode.people',
 'geonode.base',
 'geonode.layers',
 'geonode.maps',
 'geonode.proxy',
 'geonode.security',
 'geonode.social',
 'geonode.catalogue',
 'geonode.documents',
 'geonode.api',
 'geonode.groups',
 'geonode.services',
 'geonode.geoserver',
 'geonode.upload',
 'geonode.tasks')
Installed Middleware:
('django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'pagination.middleware.PaginationMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware')

Traceback:
File "/usr/lib/python2.7/dist-packages/django/core/handlers/base.py" in get_response
  112.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/usr/lib/python2.7/dist-packages/django/contrib/admin/views/decorators.py" in _checklogin
  17.             return view_func(request, *args, **kwargs)
File "/usr/lib/python2.7/dist-packages/django/views/generic/base.py" in view
  69.             return self.dispatch(request, *args, **kwargs)
File "/usr/lib/python2.7/dist-packages/account/mixins.py" in dispatch
  17.             return super(LoginRequiredMixin, self).dispatch(request, *args, **kwargs)
File "/usr/lib/python2.7/dist-packages/django/views/generic/base.py" in dispatch
  87.         return handler(request, *args, **kwargs)
File "/usr/lib/python2.7/dist-packages/django/views/generic/edit.py" in post
  205.         return super(BaseCreateView, self).post(request, *args, **kwargs)
File "/usr/lib/python2.7/dist-packages/django/views/generic/edit.py" in post
  171.             return self.form_valid(form)
File "/usr/lib/python2.7/dist-packages/account/views.py" in form_valid
  817.         signup_code.send()
File "/usr/lib/python2.7/dist-packages/account/models.py" in send
  221.         hookset.send_invitation_email([self.email], ctx)
File "/usr/lib/python2.7/dist-packages/account/hooks.py" in send_invitation_email
  15.         send_mail(subject, message, settings.DEFAULT_FROM_EMAIL, to)
File "/usr/lib/python2.7/dist-packages/django/core/mail/__init__.py" in send_mail
  50.                         connection=connection).send()
File "/usr/lib/python2.7/dist-packages/django/core/mail/message.py" in send
  276.         return self.get_connection(fail_silently).send_messages([self])
File "/usr/lib/python2.7/dist-packages/django/core/mail/backends/smtp.py" in send_messages
  94.                 sent = self._send(message)
File "/usr/lib/python2.7/dist-packages/django/core/mail/backends/smtp.py" in _send
  108.         message = email_message.message()
File "/usr/lib/python2.7/dist-packages/django/core/mail/message.py" in message
  244.         msg['Subject'] = self.subject
File "/usr/lib/python2.7/dist-packages/django/core/mail/message.py" in __setitem__
  179.         name, val = forbid_multi_line_headers(name, val, self.encoding)
File "/usr/lib/python2.7/dist-packages/django/core/mail/message.py" in forbid_multi_line_headers
  86.         raise BadHeaderError("Header values can't contain newlines (got %r for header %r)" % (val, name))

Exception Type: BadHeaderError at /account/invite_user/
Exception Value: Header values can't contain newlines (got u'Create an account on SEANODE\n' for header u'Subject')

**Comment by afabiani** [comment]
Version of GeoNode your are using?

**Comment by wen2solutions** [comment]
I used Geonode 2.4.1 version

**Comment by afabiani** [comment]
should be a dependency version issue then. Try to revert back to a earlier django-invitations version

**Comment by wen2solutions** [comment]
Is there any clue on how to revert back to the django-invitations version?

**Comment by afabiani** [comment]
ah no, wait, reading better you stacktrace it looks to me that the error is caused by a \n at the bottom of the Subject, can you double check?

**Comment by wen2solutions** [comment]
you are right..that should be mis-typo but i forgot to find the error file :(

**Comment by wen2solutions** [comment]
yes...it works well
